setup.py

```diff
@@ -30,7 +31,6 @@
         "Intended Audience :: Developers",
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
```

Comment:
Weird, we support 3.6 and 3.8+ but not 3.7?

Reply:
There is not a real issue, we will drop support for python 3.6 soon, and support only 3.8+. But I can leave python 3.7 build for now. No problem.

Reply:
Yeah better to cleanly drop 3.6 and 3.7 together when the time comes

Reply:
done